variables.tf

```diff
@@ -672,3 +672,14 @@ variable "ephemeral_storage_size" {
     error_message = "The minimum supported value is 21 GiB and the maximum supported value is 200 GiB."
   }
 }
+
+variable "task_execution_session_duration" {
```

Comment:
lets do the following please:

```hcl
variable "max_session_duration" {
  description = "Maximum session duration (in seconds) that you want to set for the specified role. Default is 3600"
  type        = number
  default     = null
}
```

Note - you'll need to rename the variable above in `main.tf`

Reply:
@bryantbiggs thanks for the prompt feedback. That should be all set.

Reply:
Also, please fix the docs by running `pre-commit run -a` and committing the changes.